src/main/java/uk/gov/pay/ledger/transaction/search/model/TransactionView.java

```diff
@@ -45,6 +47,7 @@
     private RefundSummary refundSummary;
     private SettlementSummary settlementSummary;
     private Map<String, Object> metadata;
+    private String refundedBy;
     private List<Link> links = new ArrayList<>();
 
     public TransactionView(Builder builder) {
```

Comment:
not really relevant to your changes, but this `TransactionView` should really just be a wrapper round `Transaction` and proxy calls through, rather than having to set every field again. And that would mean you wouldn't have to care about the sub-type

Reply:
Yeah, I guess it would be good to clean it up at some point.